lib/cohere-embed-v3-services-stack.ts

```diff
@@ -19,11 +19,9 @@ export class CohereEmbedV3ServicesStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props: CohereEmbedV3ServicesStackProps) {
     super(scope, id, props);
 
-    const vpcStack = new CohereEmbedVpcStack(this, `${props.resourcePrefix}-VpcStack`, {
-      ...props,
-      description: 'VPC for Cohere Embed V3.',
+    const vpc = ec2.Vpc.fromLookup(this, `${props.resourcePrefix}-VPC`, {
+      vpcId: props.vpcId,
     });
```

Comment:
**Suggestion:** When using `ec2.Vpc.fromLookup`, consider adding error handling to manage cases where the VPC ID is invalid or the VPC cannot be found.
> 🚀  **High Priority** 
 > [possible issue, importance: 8]
```suggestion
    let vpc;
    try {
      vpc = ec2.Vpc.fromLookup(this, `${props.resourcePrefix}-VPC`, {
        vpcId: props.vpcId,
      });
    } catch (error) {
      throw new Error(`Failed to lookup VPC with ID ${props.vpcId}: ${error.message}`);
    }
```